draft-westerlund-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -314,11 +316,16 @@ TLS:  Transport Layer Security
    DTLS record has been consumed. A more realistic implementation is
    two maximum DTLS record sizes.
 
-  If an implementation supports partial delivery in both the SCTP API
-  and the ULP API, and also partial processing in the DTLS/SCTP
-  implementation, then the buffering space in the DTLS/SCTP layer
-  ought to be no more than two DTLS records. In which case the MMS to
-  set is dependent on the ULP and the endpoints capabilities.
+   If an implementation supports partial delivery in both the SCTP API
+   and the ULP API, and also partial processing in the DTLS/SCTP
+   implementation, then the buffering space in the DTLS/SCTP layer
+   ought to be no more than two DTLS records. In which case the MMS to
```

Comment:
Can you elaborate on why two DTLS records is enough?

Can't you have on each incoming stream an incomplete DTLS record you can't process at the DTLS layer?

Reply:
So the comment on two DTLS records are per user message which properly should be made explicit. So you may have multiple of these, so per stream for in order delivery, and for streams with out of order delivery it becomes per message. 

It also includes an assumption that may be wrong in some implementation, that the partial delivery will only release data up to the earliest gap in the byte delivery, i.e. the data prior and including the cumulative ACK, but not ranges of data beyond a gap in the user message will be given to the layer above SCTP, and that SCTP will be responsible for holding this buffer.